forem/forem
# Remove `check_box` type from profile fields

## What type of PR is this? (check all applicable)


## Description

As @anujbhavsar96 pointed out, the `check_box` type for profile fields is not particularly useful. In fact, it was added to support some DEV functionality from back in the day, but since all these things have moved to settings and user settings this no longer seems necessary. Given that we can't really use the result of a boolean profile field in any meaningful way because we can't guarantee that exists across Forem instances I decided to remove this type completely.

## Related Tickets & Documents

n/a

## QA Instructions, Screenshots, Recordings

* Make sure you enable the profile admin feature flag: `FeatureFlag.enable(:profile_admin)`
* Go to `/admin/customization/profile_fields` and make sure field management works as expected.
* You can also go to `/settings` and verify that profile updates still work normally.

### UI accessibility concerns?

None.

## Added/updated tests?


## [Forem core team only] How will this change be communicated?
